internal/app/admin/http/cache_test.go

```diff
@@ -485,106 +485,108 @@ func testAdminServerCacheDumpHelper(t *testing.T, urls []string) {
+			req1Node := corev2.Node{
+				Id:      "test-1",
+				Cluster: "test-prod",
+			}
+			gcpReq1 := gcp.Request{
+				TypeUrl: "type.googleapis.com/envoy.api.v2.Listener",
+				Node:    &req1Node,
+			}
+			ldsRespChannel, cancelLDSWatch := orchestrator.CreateWatch(transport.NewRequestV2(&gcpReq1))
+			assert.NotNil(t, ldsRespChannel)
 
-		listener := &v2.Listener{
-			Name: "lds resource",
-		}
-		listenerAny, err := ptypes.MarshalAny(listener)
-		assert.NoError(t, err)
-		resp := v2.DiscoveryResponse{
-			VersionInfo: "1",
-			TypeUrl:     "type.googleapis.com/envoy.api.v2.Listener",
-			Resources: []*any.Any{
-				listenerAny,
-			},
-		}
-		upstreamResponseChannelLDS <- &resp
-		gotResponse := <-ldsRespChannel.GetChannel().V2
-		gotDiscoveryResponse, err := gotResponse.GetDiscoveryResponse()
-		assert.NoError(t, err)
-		assert.Equal(t, &resp, gotDiscoveryResponse)
-
-		cluster := &v2.Cluster{
-			Name: "cds resource",
-		}
-		clusterAny, err := ptypes.MarshalAny(cluster)
-		assert.NoError(t, err)
-		resp = v2.DiscoveryResponse{
-			VersionInfo: "2",
-			TypeUrl:     "type.googleapis.com/envoy.api.v2.Cluster",
-			Resources: []*any.Any{
-				clusterAny,
-			},
-		}
-		upstreamResponseChannelCDS <- &resp
-		gotResponse = <-cdsRespChannel.GetChannel().V2
-		gotDiscoveryResponse, err = gotResponse.GetDiscoveryResponse()
-		assert.NoError(t, err)
-		assert.Equal(t, &resp, gotDiscoveryResponse)
+			req2Node := corev2.Node{
+				Id:      "test-2",
+				Cluster: "test-prod",
+			}
+			gcpReq2 := gcp.Request{
+				TypeUrl: "type.googleapis.com/envoy.api.v2.Cluster",
+				Node:    &req2Node,
+			}
+			cdsRespChannel, cancelCDSWatch := orchestrator.CreateWatch(transport.NewRequestV2(&gcpReq2))
+			assert.NotNil(t, cdsRespChannel)
+
+			listener := &v2.Listener{
+				Name: "lds resource",
+			}
+			listenerAny, err := ptypes.MarshalAny(listener)
+			assert.NoError(t, err)
+			resp := v2.DiscoveryResponse{
+				VersionInfo: "1",
+				TypeUrl:     "type.googleapis.com/envoy.api.v2.Listener",
+				Resources: []*any.Any{
+					listenerAny,
+				},
+			}
+			upstreamResponseChannelLDS <- &resp
+			gotResponse := <-ldsRespChannel.GetChannel().V2
+			gotDiscoveryResponse, err := gotResponse.GetDiscoveryResponse()
+			assert.NoError(t, err)
+			assert.Equal(t, &resp, gotDiscoveryResponse)
+
+			cluster := &v2.Cluster{
+				Name: "cds resource",
+			}
+			clusterAny, err := ptypes.MarshalAny(cluster)
+			assert.NoError(t, err)
+			resp = v2.DiscoveryResponse{
+				VersionInfo: "2",
+				TypeUrl:     "type.googleapis.com/envoy.api.v2.Cluster",
+				Resources: []*any.Any{
+					clusterAny,
+				},
+			}
+			upstreamResponseChannelCDS <- &resp
+			gotResponse = <-cdsRespChannel.GetChannel().V2
+			gotDiscoveryResponse, err = gotResponse.GetDiscoveryResponse()
+			assert.NoError(t, err)
+			assert.Equal(t, &resp, gotDiscoveryResponse)
 
-		req, err := http.NewRequest("GET", url, nil)
-		assert.NoError(t, err)
+			req, err := http.NewRequest("GET", url, nil)
```

Comment:
nit: fix  indentation.

Reply:
Indentation comes from wrapping test in a `t.Run()` and running `golangci-lint`

Reply:
but this indentation is not intended, no? in that case can you manually remove the unnecessary indentation?

Reply:
I think it's intended to keep it in line with the new indentation https://github.com/envoyproxy/xds-relay/blob/21d855083faecfe9f20be54f81b7d761fbb5055f/internal/app/admin/http/cache_test.go#L567-L589

(I can still remove it, but I think it will fail unit test lint checks if I do)

Reply:
i think i have an issue in my github diff view. The indentation is fine